app/employee-app/backend/modules/authorization/utils.bal

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright (c) 2005-2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+// 
+
+# Helper function to check if user has roles.
+#
+# + requiredRoles - Required Role list
+# + userRoles - Roles list, the user has
+# + return - Allow or not
+public function checkRoles(string[] requiredRoles, string[] userRoles) returns boolean {
```

Comment:
Are we using this? In the `authorization.bal`, #61 does the same in place.

Reply:
We're using it in `service.bal` file.